packages/flame/test/game/flame_game_test.dart

```diff
@@ -698,6 +698,28 @@ void main() {
         });
       });
     });
+
+    group('pauseWhenBackgrounded:', () {
```

Comment:
Can you add some tests where it is using a widget and not calling `lifecycleStateChange` manually? There should be some other widget tests that you can take inspiration from.

Reply:
I'll add some tests, so that we can get this PR into the release.